docs/guide/blog.md

```diff
@@ -294,10 +300,12 @@ By following these steps, you have implemented all of the code required to creat
 
 Try it out! If the chain is yet not started, run `starport chain serve`.
 
-Create a post:
+Navigate to directory `cmd/blogd/` and build the go file via `go build .`.
```

Comment:
`starport chain serve` already builds the binary, there is no reason to do `go build .`.

Reply:
well it did not build it on my machine which is `Ubuntu 21.04`

Reply:
Can you please share the full instructions to reproduce that? Also, please provide the output of `starport version`.

Reply:
Sure. To make it independent of what configs are set on my machine that might impact it, I use a clean Ubuntu docker image.

`sudo docker run -it ubuntu:21.04 /bin/bash`

Inside the container:

`apt-get update`

`apt-get install -y curl golang`

`curl https://get.starport.network/starport! | bash`

Versions:
`starport version v0.17.3 linux/amd64 -build date: 2021-08-19T09:13:08Z`
`go version go1.16.2 linux/amd64`

Then:

`starport scaffold chain github.com/example/blog`

`starport chain serve`

It should have built the `blogd` command binary.

```bash
# cd cmd/blogd
ls -a
.  ..  main.go
```

Not there, but when I looked for it, I found it in the `/root/go/bin/` directory.

```bash
# find / -name "blogd"
/root/go/bin/blogd
/blog/cmd/blogd
```

So it exist but it seems like the `go/bin` path is not exported by default.
```bash
# blogd
bash: blogd: command not found
```
Do you think we should mention that in the docs that the users make sure the `go/bin` is in their default path?

Reply:
Yes, this should be already in the docs tho.

You can also user `starport c build` command with `--output` option to place the binary into a custom dir, but it is only available through the develop branch yet.

Reply:
@mojtaba-esk I think we can merge this PR, once we remove the reference to `go build`. If you feel like this is a good place to remind devs to set their PATH, we can add that.

Reply:
yes, I'll take care of it.